ntbs-integration-tests/NotificationPages/MBovisAnimalExposurePageTests.cs

```diff
@@ -217,7 +217,7 @@ public async Task AddPage_WhenModelValid_RedirectsToCollectionViewAndSavesChange
             newAnimalExposureDocument.AssertInputTextValue("MBovisAnimalExposure_YearOfExposure", "2010");
             newAnimalExposureDocument.AssertInputSelectValue("MBovisAnimalExposure_AnimalType",
                 ((int)AnimalType.Pet).ToString());
-            newAnimalExposureDocument.AssertInputSelectValue("MBovisAnimalExposure_CountryId", "3");
+            newAnimalExposureDocument.AssertInputSelectValue("selectAnimalExposureCountry", "3");
```

Comment:
Looks like adding an ID to this drop down has overridden the previous ID it was referring to? Not entirely sure I understand the behaviour but this has fixed the tests.